test/Microsoft.Azure.Devices.Edge.Test.Common/config/ModuleConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Config
+{
+    using System.Collections.Generic;
+
+    public class ModuleConfiguration
+    {
+        public IReadOnlyDictionary<string, object> Deployment { get; }
+
+        public IReadOnlyDictionary<string, object> DesiredProperties { get; }
+
+        public string Name { get; }
+
+        public ModuleConfiguration(string name, IReadOnlyDictionary<string, object> deployment, IReadOnlyDictionary<string, object> properties)
+        {
+            this.Deployment = deployment;
+            this.DesiredProperties = properties;
+            this.Name = name;
```

Comment:
check name not null or empty.

Reply:
See my previous comments about parameter validation in test code...